apps/prairielearn/src/lib/editors.ts

```diff
@@ -879,9 +882,43 @@ export class QuestionAddEditor extends Editor {
 
     const fromPath = path.join(EXAMPLE_COURSE_PATH, 'questions', 'demo', 'calculation');
     const toPath = questionPath;
+
     debug(`Copy template\n from ${fromPath}\n to ${toPath}`);
     await fs.copy(fromPath, toPath, { overwrite: false, errorOnExist: true });
 
+    if (this.files != null) {
+      debug('Remove template files when file texts provided');
+      await fs.remove(path.join(toPath, 'question.html'));
+      await fs.remove(path.join(toPath, 'server.py'));
+
+      if ('info.json' in this.files) {
+        await fs.remove(path.join(toPath, 'info.json'));
```

Comment:
Not to explode the scope of this PR, but: AFAICT, this will always create a question with the tags `mwest` and `fa17`, which is pretty undesirable. cc @mwest1066, if we don't address that here, we should address it somewhere.

Reply:
Ugh, yes, we definitely want to stop doing this. @jlevine18 can you make an issue to fix this (outside the scope of this PR)?

Reply:
See #10243.

Reply:
Thanks! Resolving this for now.